### ISSUE TITLE
Create UX/UI Design for the web app

### ISSUE THREAD

**Issue by dev-avocado** [dev-avocado/progress-tracking-system#1]
This will consist of Desktop as well as Mobile view.

Initially, you can design these pages:
1. Landing page
2. Signup page
3. Login page
4. Dashboard page

Note: Figma is the preferred software, but feel free to use other free softwares too.

**Comment by hertafui** [comment]
Can I participate in this project? @dev-avocado

**Comment by dev-avocado** [comment]
Yes @hertafui, we can also have a quick chat for discussions about UI

**Comment by hertafui** [comment]
Alright, I've sent a dm in discord (through hacktoberfest server)

**Comment by Khushi-design-20** [comment]
Hey, can i work on this?

**Comment by dev-avocado** [comment]
Hey @Khushi-design-20 
Sorry for late reply, go ahead!
